server/server.js

```diff
@@ -1,15 +1,105 @@
+require('dotenv').config();
 const express = require("express");
+const cors = require("cors");
+const { Pool } = require("pg")
+const bodyParser = require("body-parser");
+// const { v4: uuidv4 } = require("uuid");
+
 const app = express();
-const port = process.env.PORT || 5000;
 
-app.listen(port, () => console.log(`Listening on port ${port}`));
+app.use(express.json());
+app.use(cors());
+app.use(bodyParser.json());
+
+
+const db = new Pool({
+  user: process.env.DB_USERNAME,
+  host: process.env.DB_HOST,
+  database: process.env.DB_DATABASE,
+  password: process.env.DB_PASSWORD,
+  port: process.env.DB_PORT,
+});
+
+//getting all the data
+
+app.get("/", async (request, response) => {
+  try {
+    const getQuery = "SELECT id, title, link, rating FROM videos"
+
+    const result = await db.query(getQuery);
+
+    response.status(200).json({
+      "result": true, "video": result.rows
+    })
+  }
+  catch (error) {
+    response.status(400).json({
+      "result": false,
+      "message": "threre is a problem with  loading of the  page we will sort it out soon"
+    })
+  };
+});
+
+////*****DELETING VIDIO******///////
+app.delete("/:id", async (request, response) => {
+  try {
+    let videoID = +request.params.id;
+
+    if (!videoID) {
+      return response.status(400).json({ "result": "failure", "message": "there is an error!" })
```

Comment:
We might be able to be more specific than 'there is an error' do you think?
We know exactly whats wrong, theres no `id`!

Reply:
thanks, I've changed